platform/darwin/scripts/style-spec-overrides-v8.json

```diff
@@ -50,6 +53,17 @@
       "doc": "Name of image in style images to use for drawing image fills. For seamless patterns, image width and height must be a factor of two (2, 4, 8, ..., 512)."
     }
   },
+  "paint_fill-extrusion": {
+    "fill-extrusion-pattern": {
+      "doc": "Name of image in style images to use for drawing image fill-extrusions. For seamless patterns, image width and height must be a factor of two (2, 4, 8, ..., 512)."
```

Comment:
"Image fills" sort of made sense for `fill-pattern`, but I'm not sure about "image fill-extrusions". How about "for drawing the surfaces of the extrusion"?